src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -153,36 +98,69 @@
+          (let [curr-orientation (or (rf/sub [:lightbox/orientation]) orientation/portrait)
+                landscape?       (string/includes? curr-orientation orientation/landscape)
+                horizontal?      (or platform/android? (not landscape?))
+                inverted?        (and platform/ios? (= curr-orientation orientation/landscape-right))
+                screen-width     (if (or platform/ios? (= curr-orientation orientation/portrait))
+                                   (:width window)
+                                   (:height window))
+                screen-height    (if (or platform/ios? (= curr-orientation orientation/portrait))
+                                   (:height window)
+                                   (:width window))
+                item-width       (if (and landscape? platform/ios?) screen-height screen-width)]
+            (reset! common/insets-atom insets)
+            [rn/view {:style style/container-view}
+             (when-not @transparent?
+               [top-view/top-view (first messages) insets scroll-index animations landscape?
+                screen-width])
+             [rn/flat-list
+              {:ref                               #(reset! common/flat-list-ref %)
+               :key-fn                            :message-id
+               :style                             {:width (+ screen-width seperator-width)}
+               :data                              @data
+               :render-fn                         image
+               :render-data                       {:opacity-value (:opacity animations)
+                                                   :border-value  (:border animations)
+                                                   :transparent?  transparent?
+                                                   :height        screen-height
+                                                   :width         screen-width}
+               :horizontal                        horizontal?
+               :inverted                          inverted?
+               :paging-enabled                    true
+               :get-item-layout                   (fn [_ index] (get-item-layout _ index item-width))
+               :viewability-config                {:view-area-coverage-percent-threshold 50
+                                                   :wait-for-interaction                 true}
+               :shows-vertical-scroll-indicator   false
+               :shows-horizontal-scroll-indicator false
+               :on-viewable-items-changed         callback
+               :content-container-style           {:justify-content :center
+                                                   :align-items     :center}}]
+             (when (and (not @transparent?) (not landscape?))
+               [bottom-view/bottom-view messages index scroll-index insets animations
+                item-width])]))]))])
+
```

Comment:
Extra empty line. To be pedantic, the empty line means there are 2 newline characters at the end of the file, but we only need 1 to follow the Clojure Style Guide.